src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb

```diff
@@ -763,7 +763,7 @@ Namespace Microsoft.VisualStudio.Editors.MyApplication
         ''' <remarks>Caller is responsible for catching exceptions</remarks>
         Private Shared Sub SetCustomTool(ProjectItem As ProjectItem, Value As String)
             Dim ToolProperty As [Property] = GetProjectItemProperty(ProjectItem, PROJECTPROPERTY_CUSTOMTOOL)
-            Dim NamespaceProperty As [Property] = GetProjectItemProperty(ProjectItem, PROJECTPROPERTY_CUSTOMTOOLNAMESPACE)
+            'Dim NamespaceProperty As [Property] = GetProjectItemProperty(ProjectItem, PROJECTPROPERTY_CUSTOMTOOLNAMESPACE)
```

Comment:
Why?

Reply:
I wondered whether this was highlighting a bug or not. If you think it's fine, I'll remove it.

Reply:
I looked at the code, I don't think this an indicator of a bug. The namespace shouldn't be changed.